src/main/blocks.ts

```diff
@@ -5,6 +5,7 @@ import axios from "axios";
 import AdmZip from "adm-zip";
 import { sendToStatusBar } from "./logging";
 import { ncp as cpFolder } from "ncp";
+import { version } from "../../package.json";
```

Comment:
I think using `app.getVersion()` would be a good idea than manually importing package.json file